### DISCUSSION TITLE
Java 16 support?

### DISCUSSION THREAD

**Discussion by portlek** [jreleaser/jreleaser#347] [Q&A] [answered]
Is the project compatible to Java 16?

**Comment by aalmiray** [comment, marked as answer]
Could you give a bit more background to the question? Based on the contributions you've sent recently I suppose you may referring to the Gradle plugin. If so, the plugin currently targets Gradle 6.9. Both CLI and Maven plugins have been tested with Java 16.

Are there any specific failures or stacktraces that you could post?

**Reply by portlek** [reply]
Basically, I can't test my project with my local changes of jrealeaser, which is not in the early access yet, because It complains about Java version of my project which is 16. So, I decided to clone my local changes from my fork repo, but I can't build It with Java 16.
The only way to test my local changes is waiting until my changes merge.

**Reply by portlek** [reply]
For example, I want to test the early access/release feature of the project but to do that, I need GitHub Actions so I clone my fork repository then build with java 16 then it complains about java version.

**Reply by aalmiray** [reply]
OK, this is different then. One thing is for the JReleaser project to be compatible with Java 16, another for a consumer project to be compatible with Java 16.

I believe JReleaser's build requires Gradle 6.9 and Java 11 - 15 (11 is recommended).
Consumer projects should work with Java 11+ as long as you use a compatible Gradle version to build them.

**Reply by portlek** [reply]
I see, thanks.
